### DISCUSSION TITLE
[Question] Implementing a trained robot in a reinforcement learning environment.

### DISCUSSION THREAD

**Discussion by Hokite** [isaac-sim/IsaacLab#1149] [Q&A]
I'm working on reinforcement learning tasks using Isaac Lab.

Is it possible to bring a pre-trained robot into the training environment and carry out the specific learning tasks I want?

For example, when training a mobile manipulator, the manipulator is the robot I want to train, while the mobile robot is an already pre-trained robot.

Could you let me know if this is possible?

**Comment by Mayankm96** [comment]
The example here shows how to use pre-trained policies for defining action terms: https://github.com/isaac-sim/IsaacLab/tree/main/source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/navigation

That should probably help you get started :)
